scripts/aem.js

```diff
@@ -362,6 +362,7 @@ function decorateTemplateAndTheme() {
 function decorateButtons(element) {
   element.querySelectorAll('a').forEach((a) => {
     a.title = a.title || a.textContent;
+    a.target = '_blank';
```

Comment:
I'd suggest avoiding changes in `aem.js` . Also not sure, should all buttons open a new tab?

Reply:
Yes, all links are working as open in a new window in /research/equity page without login submit button. I think we can do this and about login button we can change it in the block level.

Reply:
@JiangLong2019  The concern here is in future if aem.js has any changes in boilderplate it will tough to pull those changes

Reply:
@anagarwa The aem.js can be customized based on current project requirements, do you think the link target property will make it harder to merge any further changes?

Reply:
Anyway, let's do it at the block level, I'll roll back the change.